src/main/play-doc/operation/ClusterTroubleshooting.md

```diff
@@ -61,42 +75,104 @@ sudo service conductr restart
+
+```
+http://dcos-host/#services >> Services >> conductr >> More >> Suspend >> Suspend Service
+```
+
+* Once a service has been removed the id cannot be reused. Therefore, rename your service by editing the service configuration:
+
+```
+http://dcos-host/#services >> Services >> conductr >> Edit >> <Change ID> >> Deploy Changes
+```
+
+* The previous step does not rename the existing service but instead create an additional service with the new id. Remove the old service:
+
+```
+http://dcos-host/#services >> Services >> conductr >> More >> Destroy >> Destroy Service
+```
 
-### ConductR Core node termination
+* Navigate to the new service and start it. This will start the ConductR framework instances and the executors on the Mesos agent:
+
+```
+http://dcos-host/#services >> Services >> conductr >> Scale >> <Select number of ConductR framework instances> >> Scale Service
+```
+
+## ConductR Core node termination
 
 If ConductR Core process is lost due to shutdown or crash of the `conductr` service, the bundle will be automatically replicated on a different node within the ConductR cluster.
 
 ConductR will strive to achieve configured number of replicas, and there are `3` replicas configured by default.
 
-## Restarting Bundles
-
-### ConductR Agent node termination
+## ConductR Agent node termination
 
 If a bundle dies due to a shutdown or crash of the `conductr-agent` service, the bundle will be automatically started on a different node within the ConductR cluster.
 
 The application state is not automatically restored by ConductR. When deploying a stateful bundle ensure in your application that the state is getting replicated, e.g. with [Akka Distributed Data](http://doc.akka.io/docs/akka/snapshot/scala/distributed-data.html), [Akka Cluster Sharding](http://doc.akka.io/docs/akka/snapshot/scala/cluster-sharding.html) and [Persistence](http://doc.akka.io/docs/akka/snapshot/scala/persistence.html) or distributed databases.
-     
-### Bundle termination 
-    
+
+## Bundle termination
+
 If a bundle dies of its own accord then ConductR will re-schedule it for execution elsewhere. ConductR strives to attain the scale that has been declared for a bundle (the desired scale is declared when running a bundle).
+
+## Mesos Executor termination
+
+An orphaned ConductR executor on Mesos can be terminated by logging into the Mesos agent and killing the corresponding pid. This action should only be considered in a face of disaster where the ConductR framework instances have been already removed but the executors and associated tasks are still running. Perform the following steps to find and kill the executor pid:
+
+* Retrieve the IP address of the Mesos agent on which the ConductR executor is running by using the Mesos UI:
+
+```
+http://dcos-host/mesos >> Find the executor with a task name "conductr-agent-bootstrapper" and select the <Host> address
+```
```

Comment:
Your comment seems to be related to terminating all executors on all Mesos agents. How this is done is documented under the `Restarting cluster` section. The section in which you created the comments explains the scenario how to terminate an executor on a single Mesos agent. If it is not obvious that both scenarios are described, I could cross-reference both sections with each other.

Note that the `dcos service shutdown` calls Mesos `/teardown` endpoint. So it exactly the same as calling `/teardown` directly. I think it is more convenient to use the `dcos service shutdown` command because this does not require to login into a terminal that has access to the Mesos HTTP API.